app/views/organisations/_header.html.erb

```diff
@@ -1,5 +1,68 @@
 <% if @organisation.is_no_10? %>
-  <h1 class="organisation__no10-banner organisation__margin-bottom"><%= @header.org.title %></h1>
+  <div class="govuk-width-container">
+    
+    <div class="organisation__no10-header organisation__margin-bottom">
+      <%# To do: add asset here and style and name this container as needed. Maybe add a test too? %>
+      <span class="govuk-heading-xl govuk-!-text-align-centre">
+      10
+      </span>
+      
+      <div class="organisation__no10-header__banner govuk-!-margin-bottom-6">
+
+          <%# To do: add assets here - if their ratio doesn't match the placeholder assets then you need to test them again and possibly tweak things like margins %>
+        <%= content_tag(:picture) do %>
+          <%= tag(:source, media: "(min-width: 641px)", srcset: "#{image_path('no10header-desktop.png')}") %>
+          <%= tag(:img, src: "#{image_path('no10header.png')}", alt: "") %>
+        <% end %>
+
+        <div class="organisation__no10-header__banner__content">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: "The Prime Minister's Office,<br> 10 Downing Street".html_safe,
+            heading_level: 1,
+            font_size: "l",
+            margin_bottom: 0,
+            inverse: true
+          } %>
+        </div>
+      </div>
+      <div class="organisation__no10-header__cards">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Lorem ipsum dolor sit amet",
+          heading_level: 3,
+          margin_bottom: 5
+        } %>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-one-third">
+            <%= render "govuk_publishing_components/components/image_card", {
+              brand: "prime-ministers-office-10-downing-street",
+              href: "/not-a-page",
+              heading_text: "Lorem ipsum",
+              heading_level: 3,
+              description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
```

Comment:
Should this text be in a .yml file? That's probably better than having the content in the view itself.

Reply:
@patrickpatrickpatrick We didn't discuss doing it as part of this piece of work but I've added it in anyway now. I also added tests for the yml for the controller but didn't have the time to add tests for the presenter, I've made a note of that in the PR description.